### DISCUSSION TITLE
How did you got access to the pexels api, cause they are not allowing wallpaper app ?

### DISCUSSION THREAD

**Discussion by MabudAlam** [Hash-Studios/Prism#244] [Q&A] [answered]
Pexels is not allowing wallpaper apps to use its data. I am making an app, that's why I am asking.

**Comment by codenameakshay** [comment, marked as answer]
> Pexels is not allowing wallpaper apps to use its data. I am making an app, that's why I am asking.

We have taken permission from Pexels, for this app. Please follow their official API docs to know more.
